eZ/Publish/Core/Repository/LocationResolver/LocationResolver.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Repository\LocationResolver;
+
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\API\Repository\Values\Content\Location;
+
+/**
+ * @internal For internal use by ezsystems packages
```

Comment:
"ezsystems" ceases to exist. Anyway it usually should be:
```suggestion
 * @internal For internal use by eZ Platform core packages
```